### PR TITLE
Add Expander example

### PR DESCRIPTION
**Pull request by cydanil**
Hi,
I wrote an example of `Gtk.Expander`.  

I tried to keep the tone of the text, but I'm happy to amend it :)
It seemed to me that inserting the text after the Button section had the best flow.

I've also attached a screenshot, although it looks different than the rest:
expander_example

Thanks!
Cyril

### CODE REVIEW

File: examples/expander_example.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        self.expanders = [text_expander, widget_expander]
+
+        self.show_all()
+
+    def on_expand(self, expander):
+        text_expander, widget_expander = self.expanders
+        if expander == text_expander:
+            widget_expander.set_expanded(False)
+        else:
+            text_expander.set_expanded(False)
```
Comment (sebp) [inline examples/expander_example.py:50]: I find it a little bit confusing to hide the widgets of the other expander when the expander is clicked. In particular, because usually use don't need to connect to this signal at all, because hiding/showing done automatically.

The C documentation only mentions it in case you want to create/destroy widgets on-the-fly when expanding/hiding, see https://developer.gnome.org/gtk3/stable/GtkExpander.html

File: source/expander.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+========
+
+Expanders allow to dynamically hide or show information within a window or
+dialog. An expander can take a single widget that will be displayed when
+expanded.
+
+Expanders remain expanded until clicked again. When the state of an expander
+is changed, the "activate" signal is emitted.
+
+An expander can be programmatically expanded or retracted by passing `True` or
```
Comment (sebp) [inline source/expander.txt:11]: What do you mean by "retraced"?

File: source/expander.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+`False` to :func:`Gtk.Expander.set_expanded`. Note that doing so causes the
```
Comment (sebp) [inline source/expander.txt:12]: :func: should replace by :meth:

File: source/expander.txt [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+"activate" signal to be emitted.
+
+Adding More Widgets
+-------------------
+
+More widgets, such as :Gtk.Label: or :Gtk.Button:, can be added by fitting them
```
Comment (sebp) [inline source/expander.txt:18]: I would rephrase it saying "More than one widget, such as a Label and Button, ..."

**Review by sebp — CHANGES_REQUESTED** [review]
Thanks for helping to improve the tutorial, I really appreciate your effort. I added a couple of comments that I think would improve this part.

File: source/expander.txt (hunk 2 above)
Comment (cydanil) [reply source/expander.txt:11]: I meant un-expanded, maybe "collapsed" is a better term?

File: examples/expander_example.py (hunk 1 above)
Comment (cydanil) [reply examples/expander_example.py]: Makes sense, thanks!

**Comment by cydanil** [comment]
Thank you Sebastian :)
